catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/suggestion/VerifyUsngBehaviorTest.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.suggestion;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.junit.Assert.fail;
+
+import com.google.common.collect.ImmutableList;
+import java.text.ParseException;
+import java.util.List;
+import java.util.Optional;
+import org.codice.usng4j.UtmUpsCoordinate;
+import org.codice.usng4j.impl.CoordinateSystemTranslatorImpl;
+import org.junit.Test;
+
+/**
+ * Technically this does not belong here, but these sanity checks are helpful while the feature is
+ * being developed and leftover technical debt is being worked off. These are tests that should
+ * incrementally be pushed down to the usng4j repo as bugs are fixed.
+ */
+public class VerifyUsngBehaviorTest {
+  private static final List<LatLon> EXPECTED_13_234789 =
+      ImmutableList.of(
+          new LatLon(0.01, -108.0),
+          new LatLon(8.0, -108.0),
+          new LatLon(8.0, -102.0),
+          new LatLon(0.01, -102.0));
+
+  private static final List<LatLon> EXPECTED_13_234789_2100KM_S =
+      ImmutableList.of(
+          new LatLon(-72.0, -108.0),
+          new LatLon(-64.0, -108.0),
+          new LatLon(-64.0, -102.0),
+          new LatLon(-72.0, -102.0));
+
+  private static final List<LatLon> EXPECTED_A_2347891 =
+      ImmutableList.of(
+          new LatLon(-64.0, -186.0),
+          new LatLon(-56.0, -186.0),
+          new LatLon(-56.0, -180.0),
+          new LatLon(-64.0, -180.0));
+
+  private final CoordinateSystemTranslatorImpl translator = new CoordinateSystemTranslatorImpl();
+
+  @Test(expected = IllegalArgumentException.class)
+  public void testDirectPathUtmString() {
+    // TODO - Usng4j does not find the following UTM string valid (fine due to ambiguity)
+    actual("13 234789mE 234789mN");
```

Comment:
This would actually add a bit of complexity. Take for instance: 
```
13 260499mE 7000011mN 14 260500mE 6400089mN
``` 
This input would expand to `2 ^ 2 = 4` suggestions to cover the permutations:
1. North North
2. North South
3. South North
4. South South

This rapidly spirals out of control. Three UTM coordinate inputs would be `2 ^ 3 = 8` suggestions. That ballpark number of suggestions for UTM alone would strongly interfere with meaningful suggestions from other sources. 

I cannot suggest supporting this unless absolutely necessary. We should assume north unless otherwise specified by lat band.